iron-multi-selectable.html

```diff
@@ -77,6 +78,16 @@
         (this.selectedValues != null && this.selectedValues.length);
     },
 
+    _updateAttrForSelected: function() {
+      if (!this.multi) {
+        Polymer.IronSelectableBehavior._updateAttrForSelected();
```

Comment:
You'll need to call it this way anyways though; otherwise `Polymer.IronSelectableBehavior._updateAttrForSelected` won't be called in the context of this object, but rather `Polymer.IronSelectableBehavior` itself.